mitodl/mitxonline
# Add Embedded Video to Product Page

#### Pre-Flight checklist



#### What are the relevant tickets?
Relevant tickets #80

#### What's this PR do?
Adds the following features
    - Video Url for the course video.
    - Adds video player on Course page.
    - Conditionally renders the player.
    
#### How should this be manually tested?
The following steps should be taken to test the code manually
    - Configure wagtail pages.
    - Add course page.
    - Add a video URL to course page of youtube or HLS.
    - Verify course page renders video.

#### Screenshots (if appropriate)
##### Desktop Screenshots
Video player with youtube video.
<img width="1439" alt="Screenshot 2021-08-16 at 9 15 41 AM" src="https://user-images.githubusercontent.com/87968618/129512668-56eac654-019e-497d-90fb-20eab3dadb26.png">

Video Player with HLS.
![Screenshot 2021-08-16 at 9 18 17 AM](https://user-images.githubusercontent.com/87968618/129512716-542699ea-7603-4357-a110-1504c81b5877.png)

Course page without video
<img width="1440" alt="Screenshot 2021-08-16 at 9 18 58 AM" src="https://user-images.githubusercontent.com/87968618/129512771-60904845-a717-4abc-90a3-446861bfdbde.png">

#### Mobile Screent shots

<img width="370" alt="Screenshot 2021-08-16 at 9 16 47 AM" src="https://user-images.githubusercontent.com/87968618/129512804-78b8efc8-5505-4714-ac39-6bae6987da07.png">
 
<img width="370" alt="Screenshot 2021-08-16 at 9 19 19 AM" src="https://user-images.githubusercontent.com/87968618/129512819-cbdb677f-6613-45ac-bbe1-695ac91ae6a7.png">
